src/app/client/src/app/modules/core/components/content-player-metadata/content-player-metadata.component.ts

```diff
@@ -17,6 +17,8 @@ export class ContentPlayerMetadataComponent implements OnInit, OnDestroy {
   conceptNames: any;
   filteredConcepts: any;
   conceptDataSubscription: Subscription;
+  contentCreditsData: any;
```

Comment:
Why any type for **contentCreditsData** property it should be Object or Array of Objects